src/commands/BootstrapCommand.js

```diff
@@ -521,6 +521,7 @@ export default class BootstrapCommand extends Command {
    * @param {Function} callback
    */
   symlinkPackages(callback) {
-    PackageUtilities.symlinkPackages(this.filteredPackages, this.packageGraph, this.logger, callback);
+    const { filteredPackages, packageGraph, logger } = this;
+    PackageUtilities.symlinkPackages(filteredPackages, packageGraph, logger, false, callback);
```

Comment:
Can you label the `false` please? I dislike long lists of arguments with sudden literals in the middle.
```diff
+const forceLocal = false;
const { filteredPackages, packageGraph, logger } = this;
-PackageUtilities.symlinkPackages(filteredPackages, packageGraph, logger, false callback);
+PackageUtilities.symlinkPackages(filteredPackages, packageGraph, logger, forceLocal, callback);
```